project/RELEASE-CHECKLIST.md

```diff
@@ -24,22 +24,45 @@ git remote add $GITHUBUSER git@github.com:$GITHUBUSER/docker.git
 
 ### 1. Pull from master and create a release branch
 
+#### Major releases
+
 Note: Even for major releases, all of X, Y and Z in vX.Y.Z must be specified (e.g. v1.0.0).
 
 ```bash
 export VERSION=vX.Y.Z
 git fetch origin
-git branch -D release || true
-git checkout --track origin/release
+git checkout --track origin/master
+git checkout -b release/$VERSION
+```
+
+This new branch is going to be the base for the release. We need to push it to origin so we
+can track the cherry-picked changes and the version bump:
+
+```bash
+git push origin release/$VERSION
+```
+
+When you have the major release branch in origin, we need to create the bump fork branch
+that we'll push to our fork:
+
+```bash
 git checkout -b bump_$VERSION
 ```
 
-If it's a regular release, we usually merge master.
+#### Patch releases
+
+If we have the release branch in origin, we can create the forked bump branch from it directly:
+
 ```bash
-git merge origin/master
+export VERSION=vX.Y.Z
+export PATCH=vX.Y.Z+1
+git fetch origin
+git checkout --track origin/release/$VERSION
```

Comment:
you have "release" in the name here but up on line 34 (for the major release) you don't. Is that ok?  Shouldn't they be the same?